tests/aws-cpp-sdk-core-tests/utils/stream/AwsChunkedStreamTest.cpp

```diff
@@ -81,4 +81,19 @@ TEST_F(AwsChunkedStreamTest, ShouldWorkOnSmallBuffer) {
   // Any subsequent reads will return 0 because all streams are exhausted
   amountRead = chunkedStream.BufferedRead(outputBuffer.GetUnderlyingData(), 100);
   EXPECT_EQ(0ul, amountRead);
-}
\ No newline at end of file
+}
+
+TEST_F(AwsChunkedStreamTest, ShouldWorkOnEmptyStream) {
+  StandardHttpRequest request{"www.nidia.com/juna", Http::HttpMethod::HTTP_GET};
```

Comment:
Out of curiosity, are these URLs (such as "www.nidia.com/juna") randomly generated?